integration_tests/commands/resp/dump_test.go

```diff
@@ -11,7 +10,7 @@ import (
 func TestDumpRestore(t *testing.T) {
 	conn := getLocalConnection()
 	defer conn.Close()
```

Comment:
can we add a call to a cleanup method here?

Reply:
Hey, have added a cleanup at the end of the test